app/Http/Controllers/Web/CampaignsController.php

```diff
@@ -34,7 +36,11 @@ public function __construct()
      */
     public function create()
     {
-        return view('campaigns.create')->with('causes', Cause::labels());
+        $group_type_labels = GroupType::all()->map(function ($groupType) {
+            return [$groupType->id => $groupType->name];
+        })->flatten();
```

Comment:
Let's add this as helper to the `GroupType` model per @mendelB's suggestion, to DRY and potentially to re-use in other forms (e.g. if we ever wanted to allow editing a Group's `group_type_id`).

```
    /**
     * Returns labeled list of values.
     *
     * @return array
     */
    public static function labels()
    {
        return $this::all()->map(function ($groupType) {
            return [$groupType->id => $groupType->name];
        })->flatten();
    }
```